ape_safe/multisend.py

```diff
@@ -145,7 +145,7 @@ def add(
         self,
         call,
         *args,
-        delegatecall=False,
+        delegatecall=True,
```

Comment:
So (and this is perhaps confusing), this is using delegatecall _in the downstream contract call_. not what @banteg is mentioning around actually calling the multisend itself

(a multisend is a set of downstream calls to make)

The proper solution here is to inject `operation=1` into `txn_kwargs` on `.__call__` and `.as_transaction` if and only if `"operation"` does not appear in `txn_kwargs`, AND `impersonate` is False in `txn_kwargs` (or not there at all)